core/main/src/broker/endpoint_broker.rs

```diff
@@ -787,6 +787,7 @@ fn apply_response(
                     error!("jq_compile error {:?}", e);
                 }
             }
+            trace!("mutated output {:?}", v);
```

Comment:
@pahearn73 It looks like this was removed under https://github.com/rdkcentral/Ripple/commit/5e523501d91d9890a9a57767ecd53f248c0a1b5f
and added in line 783.  Is it intentional to add this back in here as well?